treeherder/client/thclient/client.py

```diff
@@ -681,7 +682,16 @@ def _get_json(self, endpoint, timeout, project=None, **params):
 
         resp = requests.get(uri, timeout=timeout, params=params,
                             headers=self.REQUEST_HEADERS)
-        resp.raise_for_status()
+        try:
+            resp.raise_for_status()
+        except HTTPError as e:
+            response = e.response
+            logger.error("Error submitting data to %s" % response.request.url)
```

Comment:
Looking at the actual log the url is not included in the string conversion of the exception. Maybe a bug in requests? I would keep it in the log anyway.